README.md

```diff
@@ -21,6 +23,8 @@ This section documents the available CLI parameters for controlling what gets ge
  |   `--targets`          | Targets are the parts of the application that you want to be generated.
  |                        | CHOOSE ANY OF:
  |                        |   `HTTP_MODELS` - Jackson annotated data classes to represent the schema objects defined in the input.
+ |                        |   `CONTROLLERS` - Spring annotated HTTP controllers for each of the endpoints defined in the input.
+ |                        |   `SERVICES` - Default service interfaces.
```

Comment:
Should this be an option? There may be some benefit to an option for generating default service implementations, with:
```
TODO("Unable to generate implementation in the absence of an 'id' parameter")
```

Reply:
yes, I thought exactly the same. I ported it over for consistency mainly. Personally I don't find of much use getting only models and service stubs from the api spec. Should we remove it and create an issue for adding empty Service Impls?

Reply:
yes, that sounds like a good idea